hedera-node/src/test/java/com/hedera/services/txns/contract/operation/HederaBalanceOperationTest.java

```diff
@@ -0,0 +1,114 @@
+package com.hedera.services.txns.contract.operation;
+
+import org.apache.tuweni.bytes.Bytes;
+import org.hyperledger.besu.datatypes.Address;
+import org.hyperledger.besu.evm.EVM;
+import org.hyperledger.besu.evm.Gas;
+import org.hyperledger.besu.evm.account.Account;
+import org.hyperledger.besu.evm.frame.ExceptionalHaltReason;
+import org.hyperledger.besu.evm.frame.MessageFrame;
+import org.hyperledger.besu.evm.gascalculator.GasCalculator;
+import org.hyperledger.besu.evm.internal.FixedStack;
+import org.hyperledger.besu.evm.internal.Words;
+import org.hyperledger.besu.evm.worldstate.WorldUpdater;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static com.hedera.services.txns.contract.operation.HederaExceptionalHaltReason.INVALID_SOLIDITY_ADDRESS;
+import static org.hyperledger.besu.evm.frame.ExceptionalHaltReason.INSUFFICIENT_GAS;
+import static org.hyperledger.besu.evm.frame.ExceptionalHaltReason.INSUFFICIENT_STACK_ITEMS;
+import static org.hyperledger.besu.evm.frame.ExceptionalHaltReason.TOO_MANY_STACK_ITEMS;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.lenient;
+import static org.mockito.Mockito.mockStatic;
+
+@ExtendWith(MockitoExtension.class)
+class HederaBalanceOperationTest {
+
+	@Mock private GasCalculator gasCalculator;
+	@Mock private MessageFrame frame;
+	@Mock private EVM evm;
+	@Mock private Address address;
+	@Mock private Bytes bytes;
+	@Mock private WorldUpdater worldUpdater;
+	@Mock private Account account;
+	@Mock private Gas gas;
+
+	private HederaBalanceOperation subject;
+
+	@BeforeEach
+	void setUp() {
+		subject = new HederaBalanceOperation(gasCalculator);
+		givenAddress();
+		lenient().when(worldUpdater.get(any())).thenReturn(account);
+		lenient().when(frame.getWorldUpdater()).thenReturn(worldUpdater);
```

Comment:
This looks nice, unfortunately, it is not used anywhere in the whole codebase.
I would suggest switching those to
```java
given(worldUpdater.get(any())).willReturn(account);
given(frame.getWorldUpdater()).willReturn(worldUpdater);
```
and call them whenever needed.

Reply:
I think it would be better to repeat the mocks. This way tests will be more clear (not having the unnecessary mocks). Plus, it is not used anywhere in the codebase.